test/unit/ClaimProofs/addProof.js

```diff
@@ -0,0 +1,51 @@
+const { defaultSender } = require('@openzeppelin/test-environment');
+const BN = require('bn.js');
+const { assert } = require('chai');
+const accounts = require('../utils').accounts;
+
+const {
+  nonMembers: [nonMember],
+  members: [memberOne, memberTwo],
+} = accounts;
+
+describe('addProof', function () {
+
+  it('should emit an event with coverId, ipfsHash and sender when called', async function () {
+
+    const { claimProofs } = this;
+
+    const coverId = new BN(1);
+    const ipfsHash = 'mockedIpfsHash';
+    const res = await claimProofs.addProof(coverId, ipfsHash);
+    const proofAddedLog = res.logs.find(x => x.event === 'ProofAdded');
+    assert(proofAddedLog !== undefined, 'Expected a ProofAdded event');
+    assert(proofAddedLog.args[0].eq(coverId), `Expected ProofAdded event first argument to be ${coverId}`);
```

Comment:
use expectEvent for this kind of assertions like here:

https://github.com/NexusMutual/shield-mining/blob/master/test/unit/depositRewards.js#L35